randovania/data/json_data/prime1.txt

```diff
@@ -196,207 +199,421 @@ Transport to Magmoor Caverns West
 > Dock to Ruined Courtyard; Heals? False
   * Normal Door to Ruined Courtyard/Dock to Courtyard Entryway
+  > Dock to Ice Ruins West
+      Trivial
 
 > Dock to Ice Ruins West; Heals? False
   * Normal Door to Ice Ruins West/Dock to Courtyard Entryway
+  > Dock to Ruined Courtyard
+      Trivial
 
 ----------------
 Canyon Entryway
 Asset id: 55410999
 > Dock to Phendrana Canyon; Heals? False
   * Normal Door to Phendrana Canyon/Dock to Canyon Entryway
+  > Dock to Ice Ruins West
+      Trivial
 
 > Dock to Ice Ruins West; Heals? False
-  * Normal Door to Ice Ruins West/Dock to Canyon Entryway
+  * Missile Blast Shield to Ice Ruins West/Dock to Canyon Entryway
+  > Dock to Phendrana Canyon
+      Trivial
 
 ----------------
 Chapel of the Elders
 Asset id: 1086671081
 > Dock to Chapel Tunnel; Heals? False
-  * Normal Door to Chapel Tunnel/Dock to Chapel of the Elders
+  * Wave Door to Chapel Tunnel/Dock to Chapel of the Elders
+  > Pickup (Wave Beam)
+      Any of the following:
+          Missile
+          All of the following:
+              Morph Ball
+              Morph Ball Bomb or Power Bomb
+
+> Pickup (Wave Beam); Heals? False
+  * Pickup 41; Major Location? True
+  > Dock to Chapel Tunnel
+      Space Jump Boots
 
 ----------------
 Ruined Courtyard
 Asset id: 421627757
 > Dock to Courtyard Entryway; Heals? False
   * Normal Door to Courtyard Entryway/Dock to Ruined Courtyard
+  > Dock to Save Station A
+      All of the following:
+          Missile and Space Jump Boots and Morph Ball
+          Any of the following:
+              Spider Ball
+              Morph Ball Bomb and Boost Ball
 
 > Dock to Save Station A; Heals? False
-  * Normal Door to Save Station A/Dock to Ruined Courtyard
+  * Missile Blast Shield to Save Station A/Dock to Ruined Courtyard
+  > Dock to Courtyard Entryway
+      Trivial
+  > Dock to Specimen Storage
+      Space Jump Boots
+  > Dock to Quarantine Access
+      All of the following:
+          Wave Beam and Super Missile and Space Jump Boots
+          Thermal Visor or Invisible Objects (Beginner)
+  > Pickup (Energy Tank)
+      Morph Ball Bomb and Space Jump Boots and Morph Ball
 
 > Dock to Specimen Storage; Heals? False
-  * Normal Door to Specimen Storage/Dock to Ruined Courtyard
+  * Wave Door to Specimen Storage/Dock to Ruined Courtyard
+  > Dock to Courtyard Entryway
+      Trivial
+  > Dock to Save Station A
+      Missile and Space Jump Boots
 
 > Dock to Quarantine Access; Heals? False
   * Normal Door to Quarantine Access/Dock to Ruined Courtyard
+  > Dock to Courtyard Entryway
+      Trivial
+  > Dock to Save Station A
+      Missile and Space Jump Boots
+
+> Pickup (Energy Tank); Heals? False
+  * Pickup 42; Major Location? False
+  > Dock to Courtyard Entryway
+      Morph Ball
 
 ----------------
 Phendrana Canyon
 Asset id: 2718594133
 > Dock to Canyon Entryway; Heals? False
   * Normal Door to Canyon Entryway/Dock to Phendrana Canyon
+  > Pickup (Boost Ball)
+      Space Jump Boots
```

Comment:
This assumes vanilla conditions aren't possible. Should be Scan Visor or Space Jump